src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Altinn.App.Api.csproj

```diff
@@ -16,11 +16,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Altinn.Common.PEP" Version="1.0.23-alpha" />
     <PackageReference Include="Altinn.Platform.Storage.Interface" Version="2.1.12" />
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\..\..\Altinn.Common\Altinn.Common.PEP\Altinn.Common.PEP\Altinn.Common.PEP.csproj" />
```

Comment:
@TheTechArch here's you build error.. Go back to the nuget reference for PEP

Reply:
@acn-sbuad  THANK YOU! 💯  🥇